cernopendata/modules/fixtures/data/docs/cms-guide-troubleshooting/cms-guide-troubleshooting.md

```diff
@@ -1,34 +1,77 @@
-- The root password for CMS Open Data VM is `password`
+This page lists possible solutions to frequently encountered issues with CMS Open Data, for the virtual machine, for the CMSSW software and examples and for the file download and access.
 
-- In some versions of VirtualBox, the CMS Open Data VM does not open correctly. This has happened for example for VirtulBox 5.0.32, but a more recent version is OK.
+### Virtual machine
 
-- xml files do not download correctly by clicking on "Download" e.g. `http://opendata.cern.ch/record/560/files/BuildFile.xml` in [record 560](/record/560)
-Note that all files can be downloaded directly from the CMS Open Data VM terminal with wget e.g.
-```
-wget http://opendata.cern.ch/record/560/files/BuildFile.xml
-```
-
-- After the Windows update in June 2017, VirtualBox will not start. The problem has been fixed by the VirtualBox team and downloading the most recent version (5.1.22) from [the VirtualBox website](https://www.virtualbox.org/wiki/Downloads) solves the problem.
+**I have trouble trouble installing VirtualBox**
+- Read the FAQs in [the VM installation guide](/docs/cms-2011-virtual-machines-how-to-install#issue). If it still fails, please contact your local system administrator.
 
+**What is the root password for CMS Open Data VM?**
+- The root password for CMS Open Data VM is `password`.
 
-**I have trouble installing Virtualbox**
-> read the FAQs. If it still fails, please contact your local system administrator
+**The CMS Open Data VM does not open correctly**
+- In some versions of VirtualBox, it has happened that the CMS Open Data VM does not open correctly. This was the case for example for VirtulBox 5.0.32, but more recent versions from [the VirtualBox website](https://www.virtualbox.org/wiki/Downloads) have been tested and are working properly. Note that opening the CMS Open Data VM first time takes some time.
```

Comment:
Note that it can take a while to launch the CMS Open Data VM for the first time.